packages/react-instantsearch-hooks-web/src/ui/__tests__/Menu.test.tsx

```diff
@@ -285,4 +290,23 @@ describe('Menu', () => {
       'Some custom title'
     );
   });
+
+  test('renders with translations', () => {
+    const props = createProps({});
+    const { getByRole } = render(
+      <Menu
+        {...props}
+        showMore
+        translations={{
+          showMoreButtonText({ isShowingMore }) {
+            return isShowingMore ? 'Show less brands' : 'Show more brands';
+          },
+        }}
+      />
+    );
+
+    expect(
+      getByRole('button', { name: 'Show more brands' })
+    ).toBeInTheDocument();
```

Comment:
Should we re-render with `showMore=false` to test the other side of the ternary?